framework/persistence/infrastructure/seed.py

```diff
@@ -31,14 +31,21 @@ async def seed_initial_data_async(persistence: IPersistenceContext):
     persistence.add(stock_location_one)
```

Comment:
this file is up for grabs, has bug where a stock item is being generated when i don't want it to be...for some reason?

i thought it was the navigation of the shopping list but either that isn't it or it's still generating the navigation even when i set it not to